tests/lit-tests/loop_nounroll.ispc

```diff
@@ -0,0 +1,99 @@
+// Test to check '#pragma nounroll' functionality for different loop statemnts.
+
+// RUN: %{ispc} %s --target=avx2-i32x8 --nowrap -O0 --emit-llvm-text -o - | FileCheck %s -check-prefix=CHECKO0
```

Comment:
Any reason to have it for specific x86 target? This should generally be the same for any target?

Reply:
Done. Changed to host target